report/Titelblad.tex

```diff
@@ -34,15 +34,10 @@
  \end{flushright}
  \begin{flushleft}
   \textbf{Abstract:}\\
-  In the Danish educational system, the Arduino platform is often used to introduce students to computer science.
-  The Arduino's hardware is relatively simple and easy to grasp.
-  However, the Arduino language can often be difficult for novices.
-  While the Arduino language is a simplified subset of \texttt{C++}, there are still parts of the language that are difficult to comprehend for novices.
-
-  This report elaborates and explains the development of the HCL language, a programming language designed to ease the introduction to programming for students and novices, whilst also implementing high-order functionality on the Arduino.
-  HCL does this by emphasizing a resemblance to the English language.
-  A compiler for HCL was developed, as part of the project.
-  The compiler is written in Kotlin, by hand, and compiles to \texttt{C++} code.
+    More than ten thousand bird strikes take place every year at airports worldwide, causing damages for more than a billion dollars. 
+    In this report, a solution for managing bird strikes through the use of a laser-powered device using machine intelligence will be presented.
+    The solution is created using a LEGO NXT kit with the NXTOSEK real-time operating system and a raspberry pi for handling real-time image recognition, including the differences between object tracking, localization and detection.
+    The report explores the theory behind using a camera for image recognition and possibilities to implement machine intelligence for the purpose as well as how schedueling for a real-time system works, and how it can be implemented using the OSEK standard.
```

Comment:
```suggestion
    The report explores the theory behind using a camera for image recognition and possibilities for implementing machine intelligence for the purpose, as well as how schedueling for a real-time system works, and how it can be implemented using the OSEK standard.
```